.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Summary
+
+<!-- Please reference the issue this PR addresses in the following list. -->
+Addresses issue:
+
+- #
+
+## Relevant technical choices
+
+<!-- Please describe your changes. -->
+
+## Checklist
+
+- [ ] PR has `[Focus]` and `[Type]` labels.
+- [ ] PR has a milestone.
```

Comment:
These are not entirely accurate now.

* We should clarify that instead of `[Focus] ...`, it's also okay to use the `Infrastructure` label. 
* We should clarify that instead of adding a milestone for a release, it's also okay to use the `no milestone` label if this PR/issue are irrelevant of any release (e.g. not part of the release plugin ZIP).

Similar to the above, maybe it's a bit preliminary to include this here.

Reply:
Updated.